sklearn/mixture/base.py

```diff
@@ -136,7 +136,7 @@ def _initialize_parameters(self, X):
         ----------
         X : array-like, shape  (n_samples, n_features)
         """
-        n_samples = X.shape[0]
+        n_samples, _ = X.shape
```

Comment:
It just I prefer like this and it also check that the shape of X is a tuple.
It's not a big deal.